templates/front/my_checks.html

```diff
@@ -38,19 +39,22 @@
         {% include "front/my_checks_mobile.html" %}
         {% include "front/my_checks_desktop.html" %}
     {% else %}
-    <div class="alert alert-info">You don't have any checks yet.</div>
+    {% block alert_info %}
+        <div class="alert alert-info">You don't have any checks yet.</div>
+    {% endblock %}
     {% endif %}
     </div>
 </div>
-<div class="row">
-    <div class="col-sm-12">
-        <form method="post" action="{% url 'hc-add-check' %}" class="text-center">
-            {% csrf_token %}
-            <input type="submit" class="btn btn-primary btn-lg" value="Add Check">
-        </form>
+{% block add_button %}
```

Comment:
We used the blocks to avoid making another template for the checks views and failed checks view. They share a lot of content but not all like the add button feature. That block is there in my_checks.html but empty in failed_checks.html to remove it in failed_checks.html.